add FAQ: I have changed the `BaseCampWorkerMaxNum` settings, why are changes un-affected on the server? (#133)
jammsen/docker-palworld-dedicated-server: README.md

    - [How can i look into the config of my Palworld container?](#how-can-i-look-into-the-config-of-my-palworld-container)
    - [Im seeing S\_API errors in my logs when i start the container?](#im-seeing-s_api-errors-in-my-logs-when-i-start-the-container)
    - [Im using Apple silicon type of hardware, can I run this?](#im-using-apple-silicon-type-of-hardware-can-i-run-this)

>     - [I have changed the setting but seems like not affect the server.](#i-have-changed-the-setting-but-seems-like-not-affect-the-server)

Reviewer: I have changed the settings, but some changes are un-affected in the server

>     - [I have changed the setting but seems like not affect the server.](#i-have-changed-the-setting-but-seems-like-not-affect-the-server)

Reviewer: "I have changed the settings, but some changes are un-affected in the server"

Sounds/reads better

Author: ```suggestion
    - [I have changed the settings, but some changes are un-affected in the server?](#i-have-changed-the-settings-but-some-changes-are-un-affected-in-the-server)
```

### Im using Apple silicon type of hardware, can I run this?
You can try to insert in your docker-compose file this parameter `platform: linux/amd64` at the palworld service. This isnt a special fix for Apple silicon, but to run on other than x86 hosts. The support for arm exists only by enforcing x86 emulation, if that isnt to host already. Rosetta is doing the translation/emulation.

> ### I have changed the setting but seems like not affect the server.

Author: ```suggestion
### I have changed the settings, but some changes are un-affected in the server?
```

> It is a known bug, for some setting options like `BaseCampWorkerMaxNum`, changing `PalWorldSettings.ini` will not affect the server. Here is a workaround: https://github.com/legoduded/palworld-worldoptions

Reviewer: "It is a known bug ..." 
Is it known by anyone? I didnt knew this. This doesnt add anything and could be considered "less respectful" to the user that reads it.

"... for some setting options like `BaseCampWorkerMaxNum` ..."
What other variables arent working? Is it only this one? Then get rid of the "for some setting like X" because 1 of 1. The context is to vague and leaves open questions, instead of closing one.

Author: It is reported here: https://www.reddit.com/r/Palworld/comments/19f3xpk/basecampworkermaxnum_workaround_for_dedicated/

The only option reported currently is `BaseCampWorkerMaxNum`, I tried few other options but some are hard to feel of the change. Also, some options are totally not working, for example, `PalStaminaDecreaceRate`

```suggestion
It is a known bug, for example, `BaseCampWorkerMaxNum`, changing `PalWorldSettings.ini` will not affect the server. Here is a workaround: https://github.com/legoduded/palworld-worldoptions
```

Reviewer: Looking at the post there is basically only 1 option mentioned and confirmed. 
When there is only 1 option confirmed i would rather have it only formulated in a singular way.

"- [I have changed the BaseCampWorkerMaxNum settings, why are changes un-affected on the server?](#i-have-changed-the-basecampworkermaxnum-settings-why-are-changes-un-affected-on-the-server)

"It is a confirm bug, that changing `BaseCampWorkerMaxNum` in the `PalWorldSettings.ini` has no affect on the server. There are tools out there to help with this, like this one: https://github.com/legoduded/palworld-worldoptions"

Like this i would be okay with.

### Im using Apple silicon type of hardware, can I run this?
You can try to insert in your docker-compose file this parameter `platform: linux/amd64` at the palworld service. This isnt a special fix for Apple silicon, but to run on other than x86 hosts. The support for arm exists only by enforcing x86 emulation, if that isnt to host already. Rosetta is doing the translation/emulation.

### I have changed the `BaseCampWorkerMaxNum` settings, why are changes un-affected on the server?

It is a confirm bug, that changing `BaseCampWorkerMaxNum` in the `PalWorldSettings.ini` has no affect on the server. There are tools out there to help with this, like this one: <https://github.com/legoduded/palworld-worldoption>

> Please note that adding `WorldOption.sav` will breaking `PalWorldSetting.ini` and if that happens all the other settings won't be changeable anymore unless that specific tool is used everytime.

Reviewer 2: Add Bold here just to enphasize the warning, after that ok to go

Reviewer 2: on 305 line

Author: ```suggestion
**PLEASE NOTE** that adding `WorldOption.sav` will breaking `PalWorldSetting.ini` and if that happens all the other settings won't be changeable anymore unless that specific tool is used everytime.
```
